roles/edpm_nova/tasks/configure.yml

```diff
@@ -118,3 +118,70 @@
     mode: '0600'
     owner: nova
     group: nova
+
+- name: Check if compute_id exists in state_path
+  tags:
+    - configure
+    - nova
+    - adoption
+  become: true
+  ansible.builtin.stat:
+    path: /var/lib/nova/compute_id
+  register: old_compute_id
+
+- name: Copy compute_id from state_path to the config dir if present
+  tags:
+    - configure
+    - nova
+    - adoption
+  become: true
+  ansible.builtin.copy:
+    src: "/var/lib/nova/compute_id"
+    remote_src: true
+    dest: "{{ edpm_nova_config_dest }}/compute_id"
+    owner: nova
+    group: nova
+    mode: "0500"
+  when: old_compute_id.stat.exists
+
+- name: Delete compute_id from state_path
+  tags:
+    - configure
+    - nova
+    - adoption
+  become: true
+  ansible.builtin.file:
+    state: absent
+    path: "/var/lib/nova/compute_id"
+  when: old_compute_id.stat.exists
```

Comment:
so the copy and delete can just be a call to "mv"

Reply:
or something like this

```
- name: Move compute_id from state_path to the config dir if present
  ansible.posix.synchronize:
    src: "/var/lib/nova/compute_id"
    dest: "{{ edpm_nova_config_dest }}/compute_id"
    rsync_opts:
      - "--remove-source-files"
```

Reply:
I changed direction. I prefer not to move the compute_id file from the state_path (/var/lib/nova). But keep (or generate) it there and make it read-only. This greatly simplifies the logic and no adoption specific logic is needed.